scripts/build_powerplants.py

```diff
@@ -35,71 +36,68 @@
 """
 
 import logging
-import numpy as np
-import pandas as pd
 from scipy.spatial import cKDTree as KDTree
-import pycountry as pyc
 
 import pypsa
-import powerplantmatching as ppm
+import powerplantmatching as pm
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+
+def add_custom_powerplants(ppl):
+    custom_ppl_query = snakemake.config['electricity']['custom_powerplants']
+    if not custom_ppl_query:
+        return ppl
+    add_ppls = pd.read_csv(snakemake.input.custom_powerplants, index_col=0)
+    if isinstance(custom_ppl_query, str):
+        add_ppls.query(add_ppls, inplace=True)
+    return ppl.append(add_ppls, sort=False)
 
-def country_alpha_2(name):
-    try:
-        cntry = pyc.countries.get(name=name)
-    except KeyError:
-        cntry = None
-    if cntry is None:
-        cntry = pyc.countries.get(official_name=name)
-    return cntry.alpha_2
```

Comment:
Does it make sense to migrate the fallback to `official_name` also to `pm.utils`?

Reply:
yes, done in https://github.com/FRESNA/powerplantmatching/commit/411c91a5f836ae54dedcb8430da777a13dbbc6eb